android/src/main/kotlin/com/rokt/rokt_sdk/RoktWidgetFactory.kt

```diff
@@ -10,8 +10,8 @@ import io.flutter.plugin.platform.PlatformViewFactory
 class RoktWidgetFactory(private val messenger: BinaryMessenger): PlatformViewFactory(StandardMessageCodec.INSTANCE) {
     val nativeViews = mutableMapOf<Int, Widget>()
 
-    override fun create(context: Context, viewId: Int, args: Any?): PlatformView {
-        val roktWidget = RoktWidget(context, messenger, viewId)
+    override fun create(context: Context?, viewId: Int, args: Any?): PlatformView {
+        val roktWidget = RoktWidget(context!!, messenger, viewId)
```

Comment:
Context is nullable but it always is not null? so unwrapping should alway be okay I guess?

Reply:
yes it will always be non-null, I don't want to use !! here but the function return type is PlatformView which is non-nullable, so cant use null-check.

Reply:
do you know why the context become optional here? 
I think we need to see the source code to realise why they make it optional in version 3.0 but it wasn't optional in the previous versions.